### ISSUE TITLE
[Feature] Add system tray icon with options

### ISSUE THREAD

**Issue by remy-burney-pf** [KennethGomez/LiveShot#7]
Add the ability to have the application permanently in the system tray as an icon so that it's always there when you need it.
It should also have a settings menu to configure the options like HotKeys to take a screenshot, or change the image format.

![image](https://user-images.githubusercontent.com/18563478/112063227-2a4d9100-8bb5-11eb-81ac-e68daf0c483c.png)

**Comment by KennethGomez** [comment]
Yes, that's on the development roadmap in [README.md](https://github.com/KennethGomez/LiveShot#readme), I have to investigate a little to find out the best way to abord this. Thank you again 😄
